src/OpenTelemetry/Logs/LoggerSdk.cs

```diff
@@ -46,7 +46,7 @@ public override void EmitEvent(string name, in LogRecordData data, in LogRecordA
     {
         Guard.ThrowIfNullOrWhitespace(name);
 
-        string eventDomain = this.EnsureEventDomain();
+        this.EnsureEventDomain();
```

Comment:
Fine as a TODO to follow up on, but throwing here should probably be vetted with the spec. I think normally we'd not allow for exceptions in this circumstance. [Spec says](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/api.md#emit-event):

> Events require the event.domain attribute. The API MUST not allow creating an Event if the Logger instance doesn't have event.domain scope attribute.

This might be a scenario where the SDK should log, but not throw an exception.

Reply:
This seemed weird to me as well! I asked about it, here's the reply I got:

https://github.com/open-telemetry/opentelemetry-specification/pull/2768#discussion_r972447436

I'll drop a comment with that link in there for now.